### DISCUSSION TITLE
Cryptopp compilation error with clang

### DISCUSSION THREAD

**Discussion by Saikatsaha1996** [termux/termux-packages#12061] [Q&A] [answered]
Anyone can help me please?..

![IMG_20220922_150128](https://user-images.githubusercontent.com/72664192/191713255-5421fb5f-618a-4a53-ad5c-2f40c9cd94df.jpg)

**Comment by zorro** [comment, marked as answer]
`cryptopp` is already in the repo. If you really want to compile it from source check  [here](https://github.com/termux/termux-packages/blob/master/packages/cryptopp/build.sh). You'll need to install `libcpufeatures`

**Reply by Saikatsaha1996** [reply]
Thank you got it
